docs/faq.md

```diff
@@ -0,0 +1,28 @@
+Frequently Asked Questions
+==========================
+
+Here are some questions that users often have with VFS for Git, but are
+unrelated to [troubleshooting issues](troubleshooting.md).
+
+### 
+
+### Why does `gvfs clone` create a `<repo>/src` folder?
+
+VFS for Git integrates with ProjFS to keep track of changes under this `src` folder.
+Any activity in this folder is assumed to be important to Git operations. By
+creating the `src` folder, we are making it easy for your build system to
+create output folders outside the `src` directory. We commonly see systems
+create folders for build outputs and package downloads. Scalar itself creates

```

Comment:
There are several references to "Scalar" on this page.  Shouldn't these all be vfs4g ??

Reply:
Oops! I should do a better job using a text search.